src/test/java/org/opensearch/security/ssl/SecuritySSLReloadCertsActionTests.java

```diff
@@ -127,32 +127,6 @@ public void testReloadHttpSSLCertsPass() throws Exception {
         assertReloadCertificateSuccess(rh, "http", getUpdatedCertDetailsExpectedResponse("http"));
     }
 
-    @Test
-    public void testReloadHttpSSLCerts_FailWrongUri() throws Exception {
-        initClusterWithTestCerts();
-        RestHelper rh = getRestHelperAdminUser();
-
-        RestHelper.HttpResponse reloadCertsResponse = rh.executePutRequest("_opendistro/_security/api/ssl/wrong/reloadcerts", null);
-        JSONObject expectedResponse = new JSONObject();
-        // Note: toString and toJSONString replace / with \/. This helps get rid of the additional \ character.
-        expectedResponse.put("message", "invalid uri path, please use /_opendistro/_security/api/ssl/http/reload or /_opendistro/_security/api/ssl/transport/reload");
-        final String expectedResponseString = expectedResponse.toString().replace("\\", "");
-        Assert.assertEquals(expectedResponseString, reloadCertsResponse.getBody());
-    }
-
-
-    @Test
-    public void testSSLReloadFail_UnAuthorizedUser() throws Exception {
-        initClusterWithTestCerts();
-        // Test endpoint for non-admin user
-        RestHelper rh = getRestHelperNonAdminUser();
-
-        final RestHelper.HttpResponse reloadCertsResponse = rh.executePutRequest(RELOAD_TRANSPORT_CERTS_ENDPOINT, null);
-        Assert.assertEquals(401, reloadCertsResponse.getStatusCode());
-        Assert.assertEquals("Unauthorized", reloadCertsResponse.getStatusReason());
-    }
-
-
     @Test
```

Comment:
Should these tests be moved into the new SSL test class you made?

Reply:
Like with Actions tests need t be cleaned to make it more readable and better to support. I haven't touched them yet just to avoid of breaking everything.

Reply:
That makes sense. Your contribution is greatly appreciated.